(sakuli/sakuli-enterprise-forwarder#30) added checkmk spool url doc (#17)
sakuli/docs: content/enterprise_features/check.md

|`sakuli.forwarder.check_mk.piggyback_hostname`|`local`|Defines Hostname for check results, configurable for piggyback results |
|`sakuli.forwarder.check_mk.output.details`|`true`|Dis- / Enables detailed HTML output|
|`sakuli.forwarder.check_mk.section_name`|`local`|Name of the reported section (&lt;&lt;&lt;YOUR_SECTION_NAME&gt;&gt;&gt;)|

> |`sakuli.forwarder.check_mk.url.enabled`|`false`| Enables the addition of a URL to the spool file (Available from version 2.3.0) |

Reviewer: ```suggestion
|`sakuli.forwarder.check_mk.url.enabled`|`false`| Adds the URL specified in `sakuli.forwarder.check_mk.url` to the spool file (Available from version 2.3.0) |
```

> |`sakuli.forwarder.check_mk.url`|    | URL address which will be added in the spool file (Available from version 2.3.0)|

Reviewer: ```suggestion
|`sakuli.forwarder.check_mk.url`|    | URL address to add to the spool file (Available from version 2.3.0)|
```
